tests/test_errors.py

```
@@ -0,0 +1,48 @@
+        'ConnectionError: '
+        'Connection aborted while doing GET request to URL: '
+        'http://www.google.com')
+
+
+@mock.patch('httpie.core.get_response')
+def test_error_traceback(get_response):
+    exc = ConnectionError('Connection aborted')
+    exc.request = Request(method='GET', url='http://www.google.com')
+    get_response.side_effect = exc
+    with raises(ConnectionError):
+        ret = main(['--ignore-stdin', '--traceback', 'www.google.com'])
```

Ruff (0.8.2)

Local variable `ret` is assigned to but never used

Remove assignment to unused variable `ret`

(F841)